astropy/coordinates/builtin_frames/intermediate_rotation_transforms.py

```diff
@@ -22,17 +22,18 @@
 
 # # first define helper functions
 def gcrs_to_cirs_mat(time):
-    #felestial-to-intermediate matrix
-    return erfa.c2i06a(time.jd1, time.jd2)
+    #celestial-to-intermediate matrix
+    return erfa.c2i06a(time.tt.jd1, time.tt.jd2)
```

Comment:
It is not a huge deal, since the erfa routine is fairly slow on its own, but this way the scale conversion to `tt` gets done twice, which costs a few 100 micro-sec for a single-element time on my laptop. So, maybe replace with
```
time = time.tt
return erfa.c2i06a(time.jd1, time.jd2)
```
or, to speed it up even more for the case the time already is TT,
```
if time.scale != 'tt':
    time = time.tt
return erfa.c2i06a(time.jd1, time.jd2)
```